### PR TITLE
fix: reverse journal entry for multi-currency

### PR DESCRIPTION
**Pull request by AfshanKhan**
Before when making a multi-currency reverse journal entry:

Before

After the changes:

After

### CODE REVIEW

**Comment by AfshanKhan** [comment]
@Mergifyio backport version-12-hotfix

**Review by ruchamahabal — COMMENTED** [review]
@AfshanKhan Can you add a test case for Reverse Journal Entry?

**Comment by AfshanKhan** [comment]
@ruchamahabal added test case.

**Review by ruchamahabal — APPROVED** [review]

File: erpnext/accounts/doctype/journal_entry/journal_entry.py [hunk 1]
```diff
@@ -1021,3 +1021,34 @@ def make_inter_company_journal_entry(name, voucher_type, company):
 	journal_entry.posting_date = nowdate()
 	journal_entry.inter_company_journal_entry_reference = name
 	return journal_entry.as_dict()
+
+@frappe.whitelist()
+def make_reverse_journal_entry(source_name, target_doc=None, ignore_permissions=False):
```
Comment (rmehta) [inline erpnext/accounts/doctype/journal_entry/journal_entry.py:1026]: why do you need `ignore_permissions` in a whitelisted method? @AfshanKhan ? This beats the purpose

Comment (AfshanKhan) [reply erpnext/accounts/doctype/journal_entry/journal_entry.py:1026]: Understood. Will change this.

**Pull request merged**